lib/components/UnsupportedNetwork.jsx

```diff
@@ -29,9 +31,14 @@ export const UnsupportedNetwork = (props) => {
   )
 }
 
-const Network = (props) => (
-  <span className={classnames('capitalize', networkColorClassname(props.chainId))}>
-    <b>{chainIdToName(props.chainId)}</b>
-    <small className='ml-1'>{`network id: ${props.chainId}`}</small>
-  </span>
-)
+const Network = (props) => {
+  const { chainId } = props
+  const viewName = chainId && getChain(chainId)?.name
```

Comment:
Is `name` equivalent to `view` from `getChain`? Ex: `xdai` vs `xDai`

Reply:
Nope, not exactly. That's one of the problems with using the `evm-chains` package instead of our own version of it. For instance, BSC is "Binance Smart Chain Mainnet" in `evm-chains` while before our `view` would have simply been "Binance" or "BSC". We may need to inject a short name in for more control of this. There is a bunch of inconsistencies in the data (which is coming from https://chainid.network/)